ui/app/components/generated-item.js

```diff
@@ -45,7 +45,7 @@ export default Component.extend({
     deleteItem() {
       this.model.destroyRecord().then(() => {
         this.router.transitionTo('vault.cluster.access.method.item.list').followRedirects();
-        this.flashMessages.success(`${this.model.id} ${this.itemType} was deleted successfully.`);
+        this.flashMessages.success(`Successfully deleted ${this.itemType} ${this.name}.`);
```

Comment:
Ah I think this is the reason for the `undefined` - it probably should be `this.model.id`

Reply:
🦅 eyes! thank you for catching that.